### PR TITLE
UI fixes [FOGL-2769]

### PR DESCRIPTION
**Pull request by monika-Dianomic**

### CODE REVIEW

File: src/app/components/layout/navbar/navbar.component.html [hunk 1]
```diff
@@ -67,14 +67,14 @@
                 </tr>
               </table>
             </div>
-            <hr class="dropdown-divider">
+            <hr class="dropdown-divider" *ngIf="servicesRecord.length !== 0" >
```
Comment (praveen-garg) [inline src/app/components/layout/navbar/navbar.component.html]: do we need this here as well?

File: src/app/components/layout/navbar/navbar.component.html [hunk 2]
```diff
@@ -141,14 +141,14 @@
                     </tr>
                   </table>
                 </div>
-                <hr class="dropdown-divider">
+                <hr class="dropdown-divider" *ngIf="servicesRecord.length !== 0">
```
Comment (praveen-garg) [inline src/app/components/layout/navbar/navbar.component.html]: do we need this here as well?

File: src/app/components/layout/navbar/navbar.component.html [hunk 3]
```diff
@@ -192,7 +192,7 @@
                 </a>
               </div>
               <div class="dropdown-menu" id="dropdown-menu" role="menu">
-                <div class="dropdown-content">
+                <div *ngIf="servicesRecord.length !== 0" class="dropdown-content">
```
Comment (praveen-garg) [inline src/app/components/layout/navbar/navbar.component.html]: do we need this here?  

Are we missing on  `<ng-container *ngIf="!showSpinner">`

File: src/app/components/layout/navbar/navbar.component.html (hunk 1 above)
Comment (monika-Dianomic) [reply src/app/components/layout/navbar/navbar.component.html]: yes, because if there is no service list below <hr>, then there is no sense to show <hr>

**Comment by mshariq-nerd** [comment]
@monika-Dianomic  please clear the service record array on logout of the user. So that on login page old data should not be displayed in service dropdown (in forbidden state)

**Pull request merged**